client/src/components/UserSettings/userSettingsHandlers.ts

```diff
@@ -0,0 +1,49 @@
+import { ColorState as IColorState } from 'react-color';
+
+export const pickerColors = [
+  '#ff0000',
+  '#FF6900',
+  '#FCCB00',
+  '#37D67A',
+  '#308e00',
+  '#2db7f5',
+  '#004DCF',
+  '#a326f4',
+  '#FA28FF',
+  '#F78DA7',
+];
+
+export const DEFAULT_COLOR = '#308e00';
+
+export function setTagColor(e: IColorState, tagName: string, localStorageHook: (value: string) => void, storedTagColors: string | undefined) {
+  const parsedTagColors = JSON.parse(<string>storedTagColors) || {};
+  parsedTagColors[tagName] = e.hex;
+  localStorageHook(JSON.stringify(parsedTagColors));
+}
+
+export function getTagColor(tagName: string, storedTagColors: string | undefined) {
+  const parsedTagColors = JSON.parse(<string>(storedTagColors)) || {};
```

Comment:
We can remove JSON.parse and JSON.stringify here too. Also let's move getter and setter to TagSettings file.

Reply:
Removed unnecessary things. Why should we move getter and setter to `TagColor`?